src/main/java/org/galatea/starter/service/IexClient.java

```diff
@@ -32,4 +34,7 @@
   @GetMapping("/tops/last")
   List<IexLastTradedPrice> getLastTradedPriceForSymbols(@RequestParam("symbols") String[] symbols);
 
+  @GetMapping("/stable/stock/{symbol}/chart/{range}/{date}?chartByDay=true")
+  List<IexHistoricalPrices> getHistoricalPrices(@PathVariable("symbol") String symbol, @PathVariable("range") String dateRange, @PathVariable("date") String date);
```

Comment:
Are these all required parameters?

Reply:
In my current implementation, yes. I'm handling the case for passing a value of "date" to the range parameter, doing so requires us to have a third parameter for the specific date.

As part of handling non-date queries for testing (see Casey's comment above), I'm going to add a default case for when we only use symbol and range parameters.